backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/drawings.js

```diff
@@ -0,0 +1,65 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { checkQuotaIsSufficient, fileFilter } = require('./commons/revisions');
+const { createResponseCode, templates } = require('../../../../../../utils/responseCodes');
+const Yup = require('yup');
+const YupHelper = require('../../../../../../utils/helper/yup');
+const { isRevAndStatusCodeUnique } = require('../../../../../../models/revisions');
+const { respond } = require('../../../../../../utils/responder');
+const { singleFileUpload } = require('../../../../multer');
+const { statusCodes } = require('../../../../../../models/modelSettings.constants');
+const { validateMany } = require('../../../../../common');
+
+const Drawings = {};
+
+const ACCEPTED_DRAWING_EXT = ['.dwg', '.pdf'];
+const DRAWING_MAX_FILE_SIZE = 500000000;
```

Comment:
multer website says that its in bytes so the conversion gives me 500000000

![image](https://github.com/3drepo/3drepo.io/assets/71635617/6ba7548a-04ba-453e-aefc-789afbebc1e8)

the other file sizes in config have similar numbers 🤔 
![image](https://github.com/3drepo/3drepo.io/assets/71635617/311e1bc0-dfae-497c-ae25-fe98884083e9)

Reply:
I don't mean pass in MB, I mean it when I said 500MB i mean 500MiB

so it should be 500 * 1024 * 1024 instead of 500 * 1000 * 1000 (hence those existing numbers are not rounded)